tests/regression/test_mean_error.py

```diff
@@ -47,14 +58,14 @@
 def _single_target_sk_metric(preds, target, sk_fn, metric_args):
     sk_preds = preds.view(-1).numpy()
     sk_target = target.view(-1).numpy()
-    res = sk_fn(sk_preds, sk_target)
+    res = sk_fn(sk_target, sk_preds)
```

Comment:
why this switch?

Reply:
So some time back I had [this issue](https://github.com/PyTorchLightning/metrics/pull/248#issuecomment-841232277) and to resolve this, @SkafteNicki came up with [this switch](https://github.com/PyTorchLightning/metrics/pull/248#issuecomment-842976718), and since then all tests are passing.

Reply:
ok, can you and a comment/explanation because otherwise will have the same question whenever anyone will be touching

Reply:
Done!